### PR TITLE
variants/msi_z690/recovery.md: improvements

### PR DESCRIPTION
**Pull request by macpijan**
Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>

### CODE REVIEW

File: docs/variants/msi_z690/faq.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# FAQ
+
+## Will this work on my `MSI PRO Z690-A WIFI DDR5` or `MSI PRO Z690-A DDR5` ?
+
+Not yet. The build is configured for the DDR4 only. It might be extended with
+the DDR5 variant in the future. **Do not try that**.
+
+## Will this work on my `MSI PRO Z690-A DDR4` ?
+
+Although we do not own that one and cannot give promises, we believe that it
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:10]:
```suggestion
Although we do not own that one and cannot give promises without testing, we believe that it
```

File: docs/variants/msi_z690/faq.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+should work the same as the `MSI PRO Z690-A WIFI DDR4`. This is almost the
+same board, just with no WiFi card.
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:12]: @macpijan can we include that board into testing process if someone will contribute 2-3 setups?

File: docs/variants/msi_z690/faq.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+## Can i safely try this on my board?
+
+If you are afraid of bricking the board and have no means of
+[recovering from failed installation](../recovery), we **do not recommend**
+trying it out until at least the v1.0.0 is released. Recovery process is not
+suitable for inexperienced users right now. We will be trying to make it
+easier, and if that happens, we may cahnge our recommendation here.
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:20]:
```suggestion
easier, and if that happens, we may change our recommendation here.
```

File: docs/variants/msi_z690/faq.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+The other reason might be improper or interrupted installation. Please make
+sure to follow the [Installation Manual](../installation-manual) correctly.
+
+## How can I "unbrick" my board?
+
+Please follow the [recovery procedure](../recovery).
+
+## Which CPUs are supported?
+
+We can say the "supported" one is the one we have verified. It is the
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:52]:
```suggestion
We can say the "supported" one is the one we have tested. It is the
```

File: docs/variants/msi_z690/faq.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+functional, unless you have a way to get logging from the
+[Serial header](../development/#hardware-connection)) to know that it is
+actually working.
+
+12900KS is rather unknown at the moment, because it was released later and may
+require more recent microcode.
+
+## Which DDR4 memory modules are supported?
+
+We can say the "supported" one is the one we have verified. It is the
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:70]:
```suggestion
We can say the "supported" one is the one we have tested. It is the
```

File: docs/variants/msi_z690/faq.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+## How can I "unbrick" my board?
+
+Please follow the [recovery procedure](../recovery).
+
+## Which CPUs are supported?
+
+We can say the "supported" one is the one we have verified. It is the
+`Intel Core i5-12600K` and you can check that in the
+[Hardware Configuration Matrix](../hardware-matrix).
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:54]: Can we add one sentence how we tested or point to testing procedure?

File: docs/variants/msi_z690/faq.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+actually working.
+
+12900KS is rather unknown at the moment, because it was released later and may
+require more recent microcode.
+
+## Which DDR4 memory modules are supported?
+
+We can say the "supported" one is the one we have verified. It is the
+`Kingston Fury Beast KF436C17BBK4/32` part populated in all 4 slots, and you
+can check that in the [Hardware Configuration Matrix](../hardware-matrix).
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:72]: Can we add one sentence how we tested or point to testing procedure?

File: docs/variants/msi_z690/faq.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+No other memory modules or combination in slots were verified so far. This is
+on the roadmap for future releases.
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:75]: @macpijan at some point we should publish roadmap. Even rough one would be something good to stick with.

File: docs/variants/msi_z690/faq.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+Some mention about the Motherboard Memory Compatibiliy List not applying to
+Dasharo due to the differences between IBV modified code and Intel reference
+FSP
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:79]:
```suggestion
FSP.
```

File: docs/variants/msi_z690/faq.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+Maybe linking the PDFs that we had to go though when we picked the memory
+modules, since Intel maintains its own FSP compatibility list so we are
+expecting those to work. Are those public?
```
Comment (pietrushnic) [inline docs/variants/msi_z690/faq.md:83]: @miczyg1 @macpijan let's try to get it and see what rules are applied for that, aren't this information available in FSP integration guide?

**Review by pietrushnic — CHANGES_REQUESTED** [review]
@macpijan every project needs that and it is always worth to invest in such documentation, so I really appreciate you did that for this project.

File: docs/variants/msi_z690/faq.md (hunk 2 above)
Comment (macpijan) [reply docs/variants/msi_z690/faq.md:12]: You mean: if someone donates 2-3 setups for us for testing? I believe so. Even one should probably be enough.

Comment (pietrushnic) [reply docs/variants/msi_z690/faq.md:12]: @macpijan so here we should think about "how you can help" page, partially sent in previous newsletter, but what we see here, someone can donate hardware DRAM modules, CPU or mainabords.

Comment (pietrushnic) [reply docs/variants/msi_z690/faq.md:12]: We should inform people how to do that.

**Pull request merged**